docs/docs/guides/adding-metadata-key.md

```diff
@@ -0,0 +1,34 @@
+import Tabs from '@theme/Tabs';
```

Comment:
These don't seem to be rendering but getting included in the view.

Reply:
I think there is confusion here CMIIW, this page will have only one tab - `HTTP` since there is no `CLI` counterpart to it.